### DISCUSSION TITLE
Input/Output comparison across models

### DISCUSSION THREAD

**Discussion by nolandubeau** [danny-avila/LibreChat#2732] [Feature Requests & Suggestions]
Recently in independent Databricks and GCP Vertex AI demos they demonstrated the ability to have two chat UI's side by side perform the same conversation but interacting with different models.  i.e. OpenAI on the Left and Gemini on the Right.  I believe this is a very useful feature to get even more granular in testing inputs/outputs against different models since we are seeing this capability with other vendors.  The difference being is that this functionality is generally only available to devs through the vendor dev consoles as opposed to being user/consumer facing.

**Comment by danny-avila** [comment]
This is a planned feature on the roadmap!

**Reply by R-Prady** [reply]
open web UI has this feature and also inline code rendering for generate code snippets. Im glad its on the road map

**Reply by danny-avila** [reply]
I'm happy to say this feature is already live: 

https://github.com/danny-avila/LibreChat/assets/110412045/96c72d35-f183-43bd-9efe-52e24deb9710

**Reply by R-Prady** [reply]
this is dope. you are awesome. thanks a lot
